developer/deploying/platform.md

```diff
@@ -0,0 +1,179 @@
+
+Set up an SSH key pair:
+
+```sh
+# https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+#
+# create a new SSH key pair
+# prompts for filename
+# suggest "~/.ssh/launchdock" for ease
+ssh-keygen -t rsa -b 4096 -C "you@example.com"
+
+# make sure the ssh-agent is running in the background
+eval "$(ssh-agent -s)"
+
+# add your new key to the agent
+ssh-add -K ~/.ssh/<private key created above>
+
+# add your public key to Launchdock
+reaction keys add ~/.ssh/<keyname>.pub
+```
+
+## Applications
+
+An "application" is a instance of Reaction Commerce and any services, connectors, or container dependencies that are configured to deploy as one to the application environment.
+
+### create
+
+Creating an "application" configures the environment that the built application will be deployed into.
+
+```sh
+# Create and configure your new app deployment
+reaction apps create \
+  --name <appname> \
+  -e REACTION_USER="yourname" \
+  -e REACTION_AUTH="P@s5w0rd" \
+  -e REACTION_EMAIL="you@example.com" \
+  --registry  path/to/reaction.json \
+  --settings path/to/settings.json
+```
+
+#### env
+
+Configure environment variables for the application deployment.
+
+```sh
+# set/update environment variables
+# (this triggers a redeploy of your app)
+reaction env set \
+  --app <appname> \
+  -e SOME_API_KEY="ec89jmur3jim8e34" \
+  -e SOME_OTHER_THING="dj8dr34ju3r@#$" \
+  -e MAIL_URL="smtp://USERNAME:PASSWORD@NEW_HOST:PORT"
+
+# unset environment variables
+# (this triggers a redeploy of your app)
+reaction env unset --app <appname> -e SOME_API_KEY -e SOME_OTHER_THING
+
+# list your currently set environment variables
+reaction env list --app <appname>
+```
+
+**--registry** pass a `reaction.json` to the environment
+
+**--settings** pass a Meteor `settings.json` to the environment
+
+#### domains
+
+```sh
+# add a custom domain for your app
+# (first, update your DNS to point your domain at your app's default URL)
+reaction domains add --app <appname> -d mycoolshop.com
+
+# remove a custom domain from your app
+reaction domains remove --app <appname> -d mycoolshop.com
+
+#### apps list
+# list your apps and their domains
+reaction apps list
+```
+
+### deploy
+
+Submit your application for a deployment. This will begin the CI/CD/CD process. This begins a build process that is unique for every project. Expect this entire process to take around 20 minutes.
+
+-   test and build production bundle (continious integration)
+-   build container image and publish (continious delivery)
+-   deploy application cluster (continious deployment)
+
+```sh
+    # Deploy your application
+    reaction deploy --name <appname>
+```
+
+Expect to recieve an email with the completed build status (failed or deployed).
+
+#### CI/CD Configuration
+
+[Continious Integration configuration](https://docs.gitlab.com/ee/ci/) should be commited in `.reaction/ci/config.yml`.
+
+## Basic Example
```

Comment:
Basic example

Reply:
@jshimko @machikoyasuda this refers to:

-  an as of yet, unbaked optional feature with vast potential
-  Currently, a step that any project _currently_ manually required before deploy
-  a step that we have an [issue](https://github.com/reactioncommerce/launchdock/issues/43) to remove from ux 
- a step that most reaction project master users will not experience once ours is commited 

So as you can tell, I was waffling on what/how much to explain.